loader/loader.go

```diff
@@ -434,3 +435,23 @@ func (l *loader) GetAllSnapshots(c client.Client, ctx context.Context, applicati
 
 	return &snapshots.Items, nil
 }
+
+// GetSnapshotFromPipelineRunAnnotation loads from the cluster the Snapshot referenced in the given PipelineRun's annotation.
+// If the Snapshot is not found in the cluster, an error will be returned.
+func (l *loader) GetSnapshotFromPipelineRunAnnotation(c client.Client, ctx context.Context, pipelineRun *tektonv1beta1.PipelineRun) (*applicationapiv1alpha1.Snapshot, error) {
+	if snapshotName, found := pipelineRun.ObjectMeta.Annotations[tekton.SnapshotNameLabel]; found {
+		snapshot := &applicationapiv1alpha1.Snapshot{}
+		err := c.Get(ctx, types.NamespacedName{
+			Namespace: pipelineRun.Namespace,
+			Name:      snapshotName,
+		}, snapshot)
+
+		if err != nil {
+			return nil, err
```

Comment:
Missing error chaining?

Reply:
Removed this function in the newest version